source/features/code-highlight.js

```diff
@@ -0,0 +1,43 @@
+import prism from "prismjs";
+import 'prismjs/components/prism-jsx'
+import 'prismjs/components/prism-bash'
+import 'prismjs/components/prism-git'
+import 'prismjs/components/prism-typescript'
+import 'prismjs/components/prism-scss'
+
+function isSupportedLanguage(lang) {
+	const supportedLang = ["javascript", "jsx", "bash", "git", "typescript", "html", "css", "scss"]
+	return supportedLang.includes(lang);
+}
+
+export default function() {
+	const postsContent = $(".stream .stream-items > li .js-tweet-text-container > p");
+
+	postsContent.each((i,el) => {
+		const regex = /```(\w.*)([^```]+)```/g;
+		const postContent = $(el).text();
+		const capturingGroup = regex.exec(postContent);
+
+		if (capturingGroup && capturingGroup.length === 3) {
+			const code = capturingGroup[2].replace(/↵/g, "");
+			const selectedLang = capturingGroup[1].toLowerCase();
+
+			if (isSupportedLanguage(selectedLang)) {
```

Comment:
We need some aliases:

`js` => `javascript`
`sh` => `bash`
`zsh` => `bash`
`shell` => `bash`